app/Filament/Resources/AchievementResource/Pages/EditAchievement.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Resources\AchievementResource\Pages;
+
+use App\Filament\Resources\AchievementResource;
+use App\Platform\Models\Achievement;
+use Filament\Actions;
+use Filament\Resources\Pages\EditRecord;
+
+class EditAchievement extends EditRecord
+{
+    protected static string $resource = AchievementResource::class;
+
+    public static function getNavigationLabel(): string
+    {
+        return __('Edit');
+    }
+
+    public function getSubheading(): ?string
+    {
+        /** @var Achievement $record */
+        $record = $this->getRecord();
+
+        return '[' . $record->ID . '] ' . $record->Title;
+    }
```

Comment:
Editing an achievement's published state/points does not trigger the aggregate data update jobs.

Reply:
Neither does changing the associated game.

Reply:
good call, adjusted in https://github.com/RetroAchievements/RAWeb/pull/2165/commits/075fc40c293c2d729898e34a5f80828a4918be93

Introduced a new `AchievementMoved` event which dispatches game metrics updates for both the new and the original games.

Reply:
I see this spawning a job, but it fails:
```
  2024-01-26 15:50:26 App\Platform\Listeners\DispatchUpdateGameMetricsJob  RUNNING
  2024-01-26 15:50:26 App\Platform\Listeners\DispatchUpdateGameMetricsJob  5.54ms FAIL
```

Looks like you have a typo:
```
Undefined property: App\\Platform\\Events\\AchievementMoved::$origianlGame at /var/www/html/app/Platform/Listeners/DispatchUpdateGameMetricsJob.php:49)
```

Reply:
fixed in https://github.com/RetroAchievements/RAWeb/pull/2165/commits/cadd52475b5f643089765989e370895af6052f2c